src/integrationtests/resources/com/aws/greengrass/integrationtests/policy/single-character-wildcards-in-resource.yaml

```diff
@@ -0,0 +1,29 @@
+---
+services:
+  aws.greengrass.Nucleus:
+    configuration:
+      runWithDefault:
+        posixUser: nobody
+        windowsUser: integ-tester
+      logging:
+        level: "DEBUG"
+  aws.greengrass.clientdevices.Auth:
+    configuration:
+      enableSingleCharacterWildcardMatching: true
+      deviceGroups:
+        formatVersion: "2021-03-05"
+        definitions:
+          myThing:
+            selectionRule: "thingName: myThing"
+            policyName: "thingAccessPolicy"
+        policies:
+          thingAccessPolicy:
+            subscribe:
+              statementDescription: "mqtt subscribe"
+              operations:
+                - "mqtt:subscribe"
+              resources:
+                - "mqtt:topic:${iot:Connection.Thing.ThingName}/???/test/*"
```

Comment:
Are the combination of single level `?` and multi level `*` valid in the same level and should we test for it? Ex: `mqtt:topic:myThing/???*` or `mqtt:topic:myThing/*???`

Reply:
Yeah, although it's redundant---evaluation here is more regex-like than mqtt:  `*` doesn't know anything about topic levels.  

There should be test cases for this in WildcardTrieTest already (not all are passing quite yet)